src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = { items: [] };
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus(state, action) {
+      // stick to the toolkit way, as it uses the immer library to map mutating to proper react way
+      state.items = action.payload === 'Under construction' // eslint-disable-line no-param-reassign
```

Comment:
### OPTIONAL 

 - [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿